### ISSUE TITLE
Controller 'mdHead', required by directive 'mdColumn', can't be found!

### ISSUE THREAD

**Issue by amiral84** [daniel-nagy/md-data-table#303]
Hey,

I try to use md-data-table by building following html code:

    <md-table-container>
        <table md-table>
            <thead md-head>
	        <tr md-row>
		    <th md-column>Name</th>
		    <th md-column>Stats</th>
		    <th md-column>Total Amount</th>
		</tr>
	    </thead>
	    <tbody md-body>
	        <tr>
		    <td></td>
		    <td></td>
		    <td></td>
		</tr>
	    </tbody>
        </table>
    </md-table-container>

But I get following error: 

`Error: [$compile:ctreq] Controller 'mdHead', required by directive 'mdColumn', can't be found!
http://errors.angularjs.org/1.3.13/$compile/ctreq?p0=mdHead&p1=mdColumn`

I have imported all required files in index.html, injected my application with required modules. I have checked mdColumn and mdHead functions and directives from `my-data-table.js` just in case if there is typo, but I couldn't find.

How I could fix this?

**Comment by daniel-nagy** [comment]
I can't reproduce this. @FraGoTe was getting the same error in #252.

One thing you can try is moving the `mdHead` directive above the `mdColumn` directive in the source file to see if defining it first makes a difference. I don't think it will but if it does I'll make sure it gets defined first when I concatenate all the files together.

**Comment by amiral84** [comment]
Hi Daniel,

Thanks for suggestion. I tried it but it didn't help. I still get the same error.

**Comment by amiral84** [comment]
Okay, I found solution for this problem. I checked dependencies of md-data-table and I found out that it requires at least angularjs version 1.4.0 and I had 1.3.13. Now that I updated my angularjs version to most stable (1.5.0), the md-data-table renders correctly :).

It would be good idea to add notification in the description of README.md file that this module requires at least angularjs version 1.4.0.
